syntax="proto2";

import "aap_protobuf/service/bluetooth/message/BluetoothPairingMethod.proto";

package aap_protobuf.service.bluetooth.message;

message BluetoothPairingRequest
{
    required string phone_address = 1;
    required BluetoothPairingMethod pairing_method = 2;
}
